components/centraldashboard/public/components/notebook-default-card.pug

```diff
@@ -0,0 +1,25 @@
+iron-ajax#CreateDefaultNotebook(method='POST', url='/jupyter/api/namespaces/[[namespace]]/createdefault',
+    on-response='handleNotebookCreate', on-error='handleNotebookCreateError', handle-as='json', content-type='application/json')  
+iron-ajax#GetDefaultNotebook(auto, method='GET', url='/jupyter/api/namespaces/[[namespace]]/defaultnotebook',
+    last-response='{{defaultNotebook}}', on-error='handleNotebookFetchError', handle-as='json', content-type='application/json')
+paper-card#DefaultNotebookCard(heading="{{localize('notebookDefaultCard.txtDefaultNotebook')}}")
+    div.data-content
+        template(is='dom-if', if='[[_isNotebookUndefined(defaultNotebook)]]')
+            p {{localize('notebookDefaultCard.txtDataLoading')}}
+        template(is='dom-if', if='[[!_isNotebookUndefined(defaultNotebook)]]')
+            template(is='dom-if', if='[[_isNotebookEmpty(defaultNotebook)]]')
+                p {{localize('notebookDefaultCard.txtNoDefaultNotebook')}}
+                div.button-div
+                    paper-button(on-click='_createDefaultNotebook' disabled='_isCreateDisabled') {{localize('notebookDefaultCard.create')}}
+            template(is='dom-if', if='[[!_isNotebookEmpty(defaultNotebook)]]')
+                template(is='dom-if', if='[[!_isNotebookReady(defaultNotebook)]]')
+                    p {{localize('notebookDefaultCard.txtDefaultNotReady')}}
+                template(is='dom-if', if='[[_isNotebookReady(defaultNotebook)]]')
+                        p {{localize('notebookDefaultCard.name')}}: {{defaultNotebook.notebook.name}}
+                        p {{localize('notebookDefaultCard.type')}}: {{defaultNotebook.notebook.serverType}}
+                        div.button-div
+                            paper-button(on-click='_detailNotebook') {{localize('notebookDefaultCard.details')}}
+                            paper-button(on-click='[[_connectNotebook(defaultNotebook)]]') {{localize('notebookDefaultCard.connect')}}
```

Comment:
Does this button work? Im not seeing a function definition for `_connectNotebook(variable)` in the js file, just `_connectNotebook()`.